app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectedJenkinsServers.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import moment from 'moment/moment';
+import { cx } from '@emotion/css';
+import DynamicTable from '@atlaskit/dynamic-table';
+import { JenkinsPipeline, JenkinsServer } from '../../../../src/common/types';
+import {
+	mapLastEventStatus,
+	mapLastEventStatusIcons
+} from '../JenkinsServerList/ConnectedServer/ConnectedServers';
+import {
+	connectedStateCell,
+	connectedStateCellContainer, connectedStateCellEvent,
```

Comment:
👀

Reply:
God damn IntelliJ and it's sucky atuo import